### PR TITLE
feat: `avm/res/health-bot/health-bot`

### PR DESCRIPTION
**Pull request by segraef**
[avm.res.health-bot.health-bot](https://github.com/segraef/bicep-registry-modules/actions/workflows/avm.res.health-bot.health-bot.yml)

### CODE REVIEW

**Review by AlexanderSehr — CHANGES_REQUESTED** [review]
Requires uncommenting in the `/.github/CODEOWNERS` file

**Comment by segraef** [comment]
> Requires uncommenting in the `/.github/CODEOWNERS` file

done

File: avm/res/health-bot/health-bot/main.bicep [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+@description('Optional. The lock settings of the service.')
+param lock lockType
+
+@description('Optional. Array of role assignments to create.')
+param roleAssignments roleAssignmentType
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
```
Comment (AlexanderSehr) [inline avm/res/health-bot/health-bot/main.bicep:31]:
```suggestion
@description('Optional. Enable/Disable usage telemetry for module.')
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -52,7 +52,7 @@
 /avm/res/event-grid/system-topic/ @Azure/avm-res-eventgrid-systemtopic-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/event-grid/topic/ @Azure/avm-res-eventgrid-topic-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/event-hub/namespace/ @Azure/avm-res-eventhub-namespace-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/health-bot/health-bot/ @Azure/avm-res-healthbot-healthbot-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/health-bot/health-bot/ @Azure/avm-res-healthbot-healthbot-module-owners-bicep @Azure/avm-core-team-technical-bicep
```
Comment (AlexanderSehr) [inline .github/CODEOWNERS:55]: Team must be created

Comment (eriqua) [reply .github/CODEOWNERS:55]: And added to parent team as per https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

File: .github/workflows/avm.res.health-bot.health-bot.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  ###########################
+  #   Initialize pipeline   #
+  ###########################
+  job_initialize_pipeline:
+    runs-on: ubuntu-20.04
```
Comment (eriqua) [inline .github/workflows/avm.res.health-bot.health-bot.yml:47]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/CODEOWNERS (hunk 2 above)
Comment (segraef) [reply .github/CODEOWNERS:55]: @Azure/avm-res-healthbot-healthbot-module-owners-bicep
@Azure/avm-res-healthbot-healthbot-module-contributors-bicep

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**